data_steward/cdr_cleaner/manual_cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+"""
+Some new operational fields exists, that were not blacklisted in the RDR export. These rows needs to be dropped in the 
+RDR load process so they do not make it to CDR. These do not have concept_id maps. 
+The supplemental operational_pii_fields.csv shows all present PPI codes without a mapped concepts,
+indicating which should be dropped in the “drop_value” column
+"""
+import logging
+
+import bq_utils
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+LOGGER = logging.getLogger(__name__)
+
+OPERATIONAL_PII_FIELDS_TABLE = 'operational_pii_fields'
+INTERMEDIARY_TABLE = 'operation_pii_fields_to_delete'
+
+OPERATION_PII_FIELDS_INTERMEDIARY_QUERY = """
+CREATE OR REPLACE TABLE
+    `{project}.{dataset}_sandbox.{intermediary_table}` AS (
+  SELECT
+    *
+  FROM
+    {dataset}.observation
+  WHERE
+    observation_id IN (
+    SELECT
+      observation_id
+    FROM
+      `{project}.{dataset}_sandbox.{pii_fields_table}` as pii
+    JOIN
+      `{project}.{dataset}.observation` as ob
+    ON
+      (pii.observation_source_value=ob.observation_source_value)
+    WHERE
+      drop_value=TRUE))
+"""
+
+DELETE_OPERATIONAL_PII_FIELDS_QUERY = """
+DELETE
+FROM
+    {dataset}.observation
+  WHERE
+    observation_id IN (
+    SELECT
+      observation_id
+    FROM
+      `{project}.{dataset}_sandbox.{pii_fields_table}` as pii
+    JOIN
+      `{project}.{dataset}.observation` as ob
+    ON
+      (pii.observation_source_value=ob.observation_source_value)
+    WHERE
+      drop_value=TRUE)
+"""
+
+
+def get_remove_operational_pii_fields_query(project_id, dataset_id):
+    """
+
+    :param project_id: Name of the project
+    :param dataset_id: Name of the dataset where the queries should be run
+    :return:
+    """
+
+    bq_utils.load_table_from_csv(project_id=ARGS.project_id, dataset_id=ARGS.dataset_id + '_sandbox',
+                                 table_name=OPERATIONAL_PII_FIELDS_TABLE, csv_path=None, fields=None)
+
+    queries_list = []
+
+    # Save operational pii records being deleted in sandbox `dataset.intermediary_table` .
+    query = dict()
+    query[cdr_consts.QUERY] = OPERATION_PII_FIELDS_INTERMEDIARY_QUERY.format(dataset=dataset_id,
```

Comment:
Have you tried running this using just the query with the API?  Does it work successfully without explicitly setting the `destination_dataset_id` parameter?  If it does, no change is needed.

Reply:
I tried running it with the API, and it worked.